UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -21,13 +21,16 @@ public sealed class ConfigureJoinableTaskFactoryAttribute : Attribute, ITestActi
 
         public ActionTargets Targets => ActionTargets.Test;
 
+        public static readonly LoggingService LoggingService = new LoggingService() { AutoFlush = true };
+
         public ConfigureJoinableTaskFactoryAttribute()
         {
             Application.ThreadException += HandleApplicationThreadException;
         }
 
         public void BeforeTest(ITest test)
         {
+            LoggingService?.Log($"{nameof(BeforeTest)} entry");
```

Comment:
I cannot find this output in the logs!
So I must think that our test attributes are not included correctly.
This also might be the cause why `Application.ThreadException += HandleApplicationThreadException` seems to have no effect.

Reply:
Locally, it works. Might be the same issue with the bin folder structure on AppVeyor as reported by @pmiossec.

Reply:
No, this is even worse: There is output generated by the `ConfigureJoinableTaskFactoryAttribute.LoggingService` instance. Hence the assembly _is_ found and loaded.
But `BeforeTest` and `AfterTest` are not called on AppVeyor.
The most calming cause would be that the `cibuild.cmd` does not include the `AssemblyInfo.cs` files, @RussKie.

Reply:
It is not `-ci` but `-c Release`.

Reply:
Is there a tool for the configuration of MS Arcade? XML is not intended to be written by humans.

Reply:
The Arcade SDK is a set of scripts and MSBuild props and targets (https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md). 
I've been working with them in a VSCode, and inspecting binlogs. I'm not aware of any editors for the Arcade scripts, or any other "saner" way of working with it.

> The most calming cause would be that the `cibuild.cmd` does not include the `AssemblyInfo.cs` files

`cibuild` is a convenience helper, that essentially just calls `.\Build\build.ps1` with given parameters.
`.\Build\build.ps1` runs few different types of builds, depending on the input parameters - e.g. build C++ projects, build C# projects, etc.

I can't imagine `AssemblyInfo.cs` file aren't included, if they are part of the project.
You can see pretty much all aspects of a build inspecting a binlog, something like `CscCore` target that lists the compilation command and all files that were passed to the compiler.

Reply:
One way to confirm whether `AssemblyInfo.cs` is included in a build is to write some garbage in it and observe a failing build because of this error.

Reply:
![image](https://user-images.githubusercontent.com/4403806/74070442-3203b200-4a55-11ea-8fd0-c3d657b462ee.png)

Reply:
`AssemblyInfo.cs` is compiled but the contained `Attribute`s are ignored.
Just build the Release, set a breakpoint at `BeforeTest` and debug some integration test.

Have you seen the green builds of the Debug variant?

Reply:
> Just build the Release, set a breakpoint at `BeforeTest` and debug some integration test.

Release build have reduce debug symbols, so not all information may be available.

Having said that, I can hit a breakpoint in `BeforeTest`:
![image](https://user-images.githubusercontent.com/4403806/74079978-cb04ee00-4a92-11ea-98d3-c21e3844059c.png)

Reply:
I wouldn't be surprised if in Release builds these instructions get inlined or optimised in a way where it is no longer possible to hit a breakpoint.

Reply:
> I cannot find this output in the logs

It was too late in the evening. I forgot `Log(..., debugOnly = false)`.
The tests seem to pass more stably with the Debug variant. Switched back to Release.